graphqlapi/src/main/java/io/stargate/graphql/schema/schemafirst/AdminSchemaBuilder.java

```diff
@@ -0,0 +1,622 @@
+                .name("namespace")
+                .description("The namespace to deploy to.")
+                .type(nonNull(GraphQLString))
+                .build())
+        .argument(
+            newArgument()
+                .name("expectedVersion")
+                .description(
+                    "The version that those changes apply to, "
+                        + "or null if this is the first deployment.\n"
+                        + "This is used to prevent concurrent updates.")
+                .type(GraphQLString)
+                .build())
+        .argument(
+            newArgument()
+                .name("migrationStrategy")
+                .description("The strategy to update the CQL schema if necessary.")
+                .type(nonNull(MIGRATION_STRATEGY_ENUM))
+                .defaultValue(MigrationStrategy.ADD_MISSING_TABLES_AND_COLUMNS)
+                .build())
+        .argument(
+            newArgument()
+                .name("force")
+                .description(
+                    "Proceed even if the previous deployment is still marked as in progress. "
+                        + "This is used to recover manually if a previous deployment failed unexpectedly during the "
+                        + "CQL migration phase.")
+                .type(nonNull(GraphQLBoolean))
+                .defaultValue(false)
+                .build())
+        .argument(
+            newArgument()
+                .name("dryRun")
+                .description(
+                    "Just parse and validate the schema, don't deploy it or apply any changes to "
+                        + "the database. This is useful in particular in conjunction with the "
+                        + "`cqlChanges` field in the result, to evaluate the impacts on the CQL "
+                        + "data model.")
+                .type(nonNull(GraphQLBoolean))
+                .defaultValue(false)
+                .build())
+        .type(DEPLOY_SCHEMA_TYPE);
+  }
+
+  private static final GraphQLFieldDefinition DEPLOY_SCHEMA_MUTATION =
+      deploySchemaStart()
+          .name("deploySchema")
+          .description("Deploys a GraphQL schema to a namespace.")
+          .argument(
+              newArgument()
+                  .name("schema")
+                  .description("The contents of the schema.")
+                  .type(nonNull(GraphQLString))
+                  .build())
+          .build();
+
+  private static final GraphQLScalarType UPLOAD_SCALAR =
+      newScalar().name("Upload").coercing(new InputStreamCoercing()).build();
+
+  private static final GraphQLFieldDefinition DEPLOY_SCHEMA_FILE_MUTATION =
+      deploySchemaStart()
+          .name("deploySchemaFile")
+          .description(
+              "Deploys a GraphQL schema to a namespace via a file upload.\n"
+                  + "This mutation must be executed with a [multipart request](https://github.com/jaydenseric/graphql-multipart-request-spec) "
```

Comment:
Does this actually render markdown?

Reply:
Yes, per the GraphQL specification descriptions support Markdown.